editor/src/messages/portfolio/document/node_graph/node_properties.rs

```diff
@@ -60,17 +60,12 @@ pub fn expose_widget(node_id: NodeId, index: usize, data_type: FrontendGraphData
 			"Expose this parameter as a node input in the graph"
 		})
 		.on_update(move |_parameter| {
-			Message::Batched(Box::new([
-				NodeGraphMessage::ExposeInput {
-					input_connector: InputConnector::node(node_id, index),
-					set_to_exposed: !exposed,
-					start_transaction: true,
-				}
-				.into(),
-				DocumentMessage::GraphViewOverlay { open: true }.into(),
-				NavigationMessage::FitViewportToSelection.into(),
-				DocumentMessage::ZoomCanvasTo100Percent.into(),
-			]))
+			Message::Batched(Box::new([NodeGraphMessage::ExposeInput {
+				input_connector: InputConnector::node(node_id, index),
+				set_to_exposed: !exposed,
+				start_transaction: true,
+			}
+			.into()]))
```

Comment:
only zoom into the exposed node if node graph is closed (implemented in ExposeInput)